src/entities/Catalog/Item/styled.ts

```diff
@@ -0,0 +1,65 @@
+'use client';
+import { styled } from 'styled-components';
+
+import { StFlex } from '@/shared/styles/global';
+import { theme } from '@/shared/styles/theme';
+import { Button } from '@/shared/ui/Button';
+
+export const StItem = styled.article`
+  display: flex;
+  flex-direction: row;
+  min-width: 820px;
+  padding: ${theme.padding.catalogItem};
+  background: ${theme.colors.white};
+  border-radius: ${theme.borderRadius.catalogItem};
+  box-shadow: 0 4px 4px 0 ${theme.colors.boxShadow};
+  gap: 52px;
```

Comment:
```suggestion
  column-gap: 52px;
```
И было бы клёво перевести все отступы в rem

Reply:
У нас в проекте уже в px, давайте единый стиль придерживаться. Можно на собрании решить по поводу отступов в em, но т.к. у нас размер шрифтов скачет, имхо, это сложнее.